wordpress-mobile/WordPress-Android
# Material 3: Blaze

This PR updates the Blaze feature to Material3, affecting these three areas:

* `BlazeOverlayFragment` - shown when "Learn more" is selected from the Blaze card on the My Site tab
* `BlazePromoteParentActivity` - shown when tapping the Blaze card on the My Site tab
* `CampaignListingFragment` - shown when tapping More on the My Site tab, then tapping Blaze

To test:

* Switch to a site that has at least one Blaze campaign
* On the main My Site tab, tap the ☰ on the Blaze card
* Select "Learn more"
* Verify the screen looks correct
* Return to the My Site tab and tap the Blaze card
* Verify the screen looks correct (note that the UI is in a WebView)
* Return to the My Site tab and tap More
* Tap Blaze and verify the screen looks correct
* Switch to a site that doesn't have any Blaze campaigns and verify the feature works as expected

**Note:** I feel like this feature could use some UI work, but much of it is hosted in a WebView, the content of which isn't in our control.


https://github.com/user-attachments/assets/c67160e2-fe46-481e-9dc7-55ce3708df39
